example/lib/example.g.dart

```diff
@@ -19,7 +19,7 @@ Person _$PersonFromJson(Map<String, dynamic> json) {
 }
 
 Map<String, dynamic> _$PersonToJson(Person instance) {
-  var val = <String, dynamic>{
+  final val = <String, dynamic>{
```

Comment:
Just revert all changes to `example` – we'll have to update it later - https://travis-ci.org/dart-lang/json_serializable/jobs/436810474#L551